puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/foreman-nightly-deb-pipeline.yaml

```diff
@@ -0,0 +1,11 @@
+- job:
+    name: foreman-nightly-deb-pipeline
+    project-type: pipeline
+    triggers:
+      - timed: 'H 07 * * 1-5'
+    dsl:
+      !include-raw:
+        - pipelines/release/foreman-deb.groovy
```

Comment:
Doesn't this also need `release.groovy` for the deb pushing?

Reply:
I could have bet it has. Fixing.